orka_vector_api/helper/gdal_helper.py

```diff
@@ -18,66 +19,57 @@ def _get_geopackage_cmd(filename, layername, sql, host=None, port=None, database
     return cmd
 
 
-def create_geopackage(data_id, layer_sqls, e, db_props=None, gpkg_path=''):
-    filename = os.path.abspath(os.path.join(gpkg_path, data_id + '.gpkg'))
-    print(f'creating geopackage for {data_id}, {db_props}, {gpkg_path}')
-
-    for layername, layer_sql in layer_sqls:
-        if e.isSet():
+def _create_gpkg(data_id, bbox, timeout_e=None, error_e=None, db_props=None, gpkg_path='', layers_path=''):
+    file_name = os.path.abspath(os.path.join(gpkg_path, data_id + '.gpkg'))
+    layer_sqls = _get_layer_sqls(layers_path)
+    for layer_name, layer_sql in layer_sqls.items():
+        if timeout_e is not None and timeout_e.isSet():
             break
-        cmd = _get_geopackage_cmd(filename, layername, layer_sql, **db_props)
-        # app.logger.info(cmd)
+        gpkg_sql = _get_gpkg_sql(layer_sql, bbox)
+        gpkg_sql_escaped = _escape_sql(gpkg_sql)
+        print(gpkg_sql_escaped)
```

Comment:
should this be a logging statement?

Reply:
As we run this in a thread, we do not have access to the application. So we have to add a different logging mechanism here